### ISSUE TITLE
Does this LPA is complete 

### ISSUE THREAD

**Issue by mohammadmahdi255** [Truphone/LPAdesktop#11]
hello i want to implement a SMDP server for eSIM I want to know does this LPAd is capable of download profile from SMDP server?

**Comment by panda2134** [comment]
Yes it is. The webcam library is sometimes causing problems but you can comment those code out.
